docs/react-native-devtools.md

```diff
@@ -0,0 +1,99 @@
+---
+id: react-native-devtools
+title: React Native DevTools
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+React Native DevTools is our new debugging experience featuring an end-to-end rewrite of our debugger stack. It aims to be more deeply integrated and fundamentally more reliable than previous debugging methods in React Native.
+
+![React Native DevTools opened to the "Welcome" pane](/docs/assets/debugging-rndt-welcome.jpg)
+
+React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Android Studio and Xcode (see [Debugging Native Code](http://localhost:3000/docs/next/debugging-native-code)).
+
+<details>
+<summary>**💡 Compatibility** — released in 0.76</summary>
+
+React Native DevTools supports all React Native 0.76 apps (and onwards) running Hermes. It replaces the previous Flipper, Experimental Debugger, and Hermes debugger (Chrome) entry points.
```

Comment:
```suggestion
React Native DevTools supports all React Native apps running Hermes. It replaces the previous Flipper, Experimental Debugger, and Hermes debugger (Chrome) entry points.
```
The versioning is part of the website. Users reading docs for 0.75 will not see this page, so it's clear that is valid from 0.76 and onward

Reply:
I think it's super helpful to reinforce this, particularly as we risk confusion with the 0.73+ Experimental Debugger for this feature.

Re: the temporal point in general — I want to be specific if possible, for the above reason. We will refresh these docs continually during next year.

Reply:
> We will refresh these docs continually during next year.

I hope so, but these bits are easy to forget. I will remove them now to avoid issue in the future. I don't believe that it is helpful to keep them.

Reply:
Will keep the "released in 0.76" hook in the callout title and drop these inner ones 👍🏻